### PR TITLE
Traitor variety (take 3)

### PR DESCRIPTION
**Pull request by TokenStyle**
wololo
making this for https://github.com/space-wizards/space-station-14/pull/30198

### CODE REVIEW

**Comment by TokenStyle** [comment]
> Personally I think traitor variety should actually influence the types of objectives different traitors get

It can be implemented with some effort and game design, sounds great that some "boss" gives you objectives that they prefer, but personally thinking in gameplay loop, will it make such a big influence on player's pro-social, interactive part? And we have to do more work and code/design support in the future to achieve and maintain this feat.
Atm in doc:

> “Now every traitor have a "boss", that asks you complete some objective. No matter what boss you have, you always have random objectives.”

That's stand for *Seriously Silly*, *Player Interaction*, *Player Agency* in [design principles](https://docs.spacestation14.com/en/space-station-14/core-design/design-principles.html). When traitors don't have in-code restrictions player's would get new and never repeatetive pro-social experience, for example some amazing situations you will get:
- ARC objective's butcher Ian? It's hilarius, would it be funny to tell other agents about your boss objective's? Is it give you new and interesting interaction to you? Yes.
- Your boss Donc corp sends you as your partner Waffle corp agent. These megacorps are against each other, but you now have to cooperate to achieve your objectives, how will you interact with that agent, what will change? Or objectives a priority so as not to cause losses to the Donc megacorp? Now you talk with your companion! (you *don't* kill each other, because notes are *only* recommendations, not strict rules)
- Gorlax Marauders sends you just to steal a dog? Take the gun, enter the room - take that dog using brute force! When enter you say "For Gorlax Marauders!" while you hold the dog in your arms! Silly and funny!
- You are in legal trouble and have to steal HoS docs? You will stealthy enter the brig and try to steal what you want. You probably don't want to deal with sec, or you will intensionally break the law and become prisoner with big plans!

And the important one, traitors are both, like still part of the Syndicate and a traitor, and have only "in words" restrictions, not real one. These notes are by your boses, but you make the final decision, and you not have to take seriously notes at all. Traitors do everything as they see fit.
> Allegiances and additional notes are recommendations. You do not have to strictly follow them.

**TL;DR:** adding just pure text, will encourage players interact, talk, communicate with each other by giving "initial kick". Like atm about what agents could talk about? Pretty much nothing except their objectives, but with this concept players will get more room for amazing interaction!

**Comment by TokenStyle** [comment]
Also the feat has already been developed by amazing dev @whateverusername0 , after concept's approval, pressing few buttons and the feat will appear in the game, following the concept which I described it in the design document

**Comment by ElPidor** [comment]
waiting for merge.... 
syndicate hq ewe help
